### DISCUSSION TITLE
does clickhouse-backup support restore to remote server？

### DISCUSSION THREAD

**Discussion by lddlww** [Altinity/clickhouse-backup#645] [Q&A] [answered]
for exmaple , target clickhouse is built in cloud，how can i restore local data to cloud clickhouse using clickhouse-backup?

**Comment by Slach** [comment, marked as answer]
No, clickhouse-backup execute ALTER TABLE ... ATTACH PART statement during restore, so it require download backuped data to the same disk where clickhouse-server store data

only schema could be restored on remote clickhouse server, without download to the same disk
